cmd/geth/wemixcmd.go

```diff
@@ -297,14 +290,14 @@ func loadGenesisConfig(r io.Reader) (*genesisConfig, error) {
 	}
 
 	if len(config.Accounts) == 0 || len(config.Members) == 0 {
-		return nil, fmt.Errorf("At least one account and node are required.")
+		return nil, fmt.Errorf("at least one account and node are required")
 	}
 
 	bootnodeExists := false
 	for _, m := range config.Members {
 		// to conforming form to avoid checksum error
 		if !(len(m.Id) == 128 || len(m.Id) == 130) {
-			return nil, fmt.Errorf("Not a node id: %s\n", m.Id)
+			return nil, fmt.Errorf("not a node id: %s", m.Id)
```

Comment:
I think that just checking the size to check the correct format is not enough.

		if !strings.HasPrefix(m.Id, "0x") {
			m.Id = "0x" + m.Id
		}
		bytes, err := hexutil.Decode(m.Id)
		if err != nil {
			return nil, fmt.Errorf("invalid node id: %s, error: %s", m.Id, err)
		}
		if len(bytes) != 64 {
			return nil, fmt.Errorf("invalid node id: %s, error: size error", m.Id)
		}